packages/wrangler/src/create-worker-preview.ts

```diff
@@ -113,7 +113,7 @@ async function createPreviewToken(
       : `/accounts/${accountId}/workers/scripts/${scriptId}/edge-preview`;
 
   const mode: CfPreviewMode = ctx.zone
-    ? { routes: ["*/*"] } // TODO: should we support routes here? how?
+    ? { routes: [...(ctx.zone.routes ?? "*/*")] }
```

Comment:
oh so that's what it what as there for! Did this fix the secrets problem?

Reply:
This will fix the problem... as long as there is no wildcard in the route 🤷 and also as long as there is not another worker that was published with the same route.

Reply:
So we were not sure if the problem is actually that EWC no longer supports wildcards, which might be why "*/*" is no longer working...